src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user?: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return user ? (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
```

Comment:
Destructure user

Reply:
@silchyn i cant destructure it since `user `could be undefind)